Cargo.toml

```diff
@@ -115,6 +116,7 @@ syn = { version = "2.0.57", features = [
     "printing",
 ] }
 tera = { version = "1.19.1" }
+tempfile = { version = "3.10.1" }
```

Comment:
nit: alphabetical sorting.

Reply:
Done in https://github.com/NomicFoundation/slang/pull/909/commits/d198f3faf7964f6eb5a3db430a38fea17b911ac6